tanner/tests/test_base.py

```diff
@@ -138,7 +137,6 @@ def test_emulate_type_1(self):
         detection = self.loop.run_until_complete(self.handler.emulate(data, self.session))
         assert_detection = {'name': 'index', 'order': 1, 'type': 1, 'version': tanner_version}
         self.assertEqual(detection, assert_detection)
-        self.handler.set_injectable_page.assert_not_called()
```

Comment:
Why did you remove this?

Reply:
`assert_not_called()` is not available in version `3.5`. It was working in `3.6.8`

Reply:
Can you try [this](https://stackoverflow.com/questions/12187122/assert-a-function-method-was-not-called-using-mock)?

Reply:
Also, if this works then add a TODO to update when the python version is upgraded.

Reply:
Also, I think you lack the habit of googling before raising concerns. For this, I get the above link as the second result by just typing `python mock assert not called `

Reply:
Honestly, its not liked that. It just got very messy for me bcoz of working on too many files at once. But, I will be more careful from now.